clarity/src/vm/docs/mod.rs

```diff
@@ -3135,7 +3135,7 @@ mod test {
     fn test_examples() {
         // Execute test examples against the latest version of Clarity
         let apis = make_all_api_reference();
-        let token_contract_content = include_str!("../../../../sample-contracts/tokens.clar");
+        let token_contract_content = include_str!("../../../../sample/contracts/tokens.clar");
```

Comment:
I know you didn't introduce this here, but can we do something cleaner and less fragile than all these dots and slashes? There is no Cargo environment variable which contains the workspace root, but you can get it like [this](https://stackoverflow.com/a/74942075). How about we wrap this in a `LazyLock` in stacks-common/util?

Reply:
Is `LazyLock` necessary?  This code doesn't even compile except for during tests.

Reply:
We resolved this a little differently, see 2d7f58ac5ea853c58c9234c3af98480859ec46a6

Reply:
we'll have to resolve situations like this a little differently due to the use of `include_str!` and will be addressed in a separate PR